cpp/ql/test/library-tests/dataflow/dataflow-tests/dataflow-consistency.expected

```diff
@@ -1,5 +1,11 @@
 uniqueEnclosingCallable
+| test.cpp:864:44:864:58 | {...} | Node should have one enclosing callable but has 0. |
+| test.cpp:864:47:864:54 | call to source | Node should have one enclosing callable but has 0. |
+| test.cpp:872:46:872:51 | call to source | Node should have one enclosing callable but has 0. |
+| test.cpp:872:53:872:56 | 1 | Node should have one enclosing callable but has 0. |
 uniqueCallEnclosingCallable
+| test.cpp:864:47:864:54 | call to source | Call should have one enclosing callable but has 0. |
+| test.cpp:872:46:872:51 | call to source | Call should have one enclosing callable but has 0. |
```

Comment:
Why are these suddenly showing up, while we don't have these for other globals for which we already had tests?

Reply:
No idea. Since this is AST dataflow I didn't really bother to investigate this.

Reply:
Ah, this is for AST dataflow, I missed that. Should have had a better look at the file names first. In that case, it seems safe to ignore.

Reply:
Note that line 864 is:
```
static const int global_array_dynamic[] = { ::source() };
```
and line 872 is:
```
static const int* global_pointer_dynamic = source(true);
```
So it's very possible that the code for these syntactic constructs simply was never generalized to handle these things appearing in global scope in AST dataflow (since AST dataflow never actually implementer proper support for global variables). Similarly, line 872 is: